### PR TITLE
Restrict Contributors to 14 value and show all on github link

### PR DESCRIPTION
**Pull request by ashishnagpal2498**
Restrict Contributors to 14. Kindly tell the link of viewing all the contributors. For now github link has been attached to it.

### CODE REVIEW

**Comment by ashishnagpal2498** [comment]
Issue fix #585 . Working on Code quality

**Comment by ashishnagpal2498** [comment]
Issue fix #585 . Working on Code quality

**Comment by ashishnagpal2498** [comment]
> 
> 
> 
> 
> Having a heading as `Our Contributors` and mentioning below `+14 contributors` is looking a little odd to me.
> My suggestion is directly writing `+14 contributors` in the heading replacing `Our contributors`.
> 
> @divyanshu-rawat what do you think?
Yes that seem bit odd to me too. But that is because of the font body has. Should I change the font to basic one ?

**Comment by divyanshu-rawat** [comment]
> > 
> > Having a heading as `Our Contributors` and mentioning below `+14 contributors` is looking a little odd to me.
> > My suggestion is directly writing `+14 contributors` in the heading replacing `Our contributors`.
> > @divyanshu-rawat what do you think?
> > Yes that seem bit odd to me too. But that is because of the font body has. Should I change the font to basic one ?

@ashishnagpal2498 can you also do changes as per @debck suggestion :)

**Comment by ashishnagpal2498** [comment]
@divyanshu-rawat @gittysachin .Thank you for such a great guidance. Sure , will work on it.

File: src/ignitus-Team/Components/Team.tsx [hunk 1]
```diff
@@ -38,7 +43,7 @@ const Team = ({
     );
   }
 
-  const contributors = presets.map((item: GitHubDataType) => (
+  const contributors = presets.map((item: GitHubDataType, index: number) => (
```
Comment (divyanshu-rawat) [inline src/ignitus-Team/Components/Team.tsx]: please remove `index: number` its no longer necessary.

**Pull request merged**